tests/test_calculator.cc

```diff
@@ -567,6 +567,146 @@ namespace rascal {
     }
   }
 
+  /* ---------------------------------------------------------------------- */
+  /**
+   * Utility fixture used to compare representations with sparsification
+   */
+
+  struct SparsificationSphericalInvariantsFixture {
```

Comment:
I like this because it's more self-contained than some of the other test fixtures; no need to go looking for dependencies in other files to figure out the data and behaviour contained in this class. Eliminating the inheritance hierarchy was particularly helpful in this regard.

Reply:
ok so at some point we should probably spend some time to converge the tests to a format similar to this one.